src/tests/ftest/pool/create_all_vm.py

```diff
@@ -75,10 +80,52 @@ def test_one_pool(self):
                 "Invalid SMD size: want=0, got={tier_bytes[1]}")
 
         self.log.info("Checking size of available storage")
-        self.scm_avail_bytes = self.get_pool_available_bytes()
+        self.scm_avail_bytes = self.get_available_bits()
         self.assertEqual(0, self.scm_avail_bytes,
                 "Invalid SCM size: want=0, got={}".format(self.scm_avail_bytes))
 
+    def test_rank_filter(self):
+        """Test the creation of one pool with filtering the rank to use
+
+        Test Description:
+            Create a pool with all the capacity of some servers. Verify that the pool created
+            effectively used all the available storage of the selected servers and the non selected
+            servers keep their available storage.
+
+        :avocado: tags=all,pr
+        :avocado: tags=vm
+        :avocado: tags=pool,pool_create_all
+        :avocado: tags=pool_create_all_rank_filter_vm,test_pool_rank_filter
+        """
+        self.log.info("Test  basic pool creation with full storage")
+
+        ranks = [rank for rank, _ in enumerate(self.hostlist_servers)]
+        ranks_used = ranks[:(len(ranks) // 2)]
+        ranks_unused = ranks[(len(ranks) // 2):]
+        scm_usable_bytes = self.get_available_bits(ranks_used)
+        scm_unused_bytes = self.scm_avail_bytes - scm_usable_bytes
+
+        create_time = self.create_one_pool(ranks_used)
+        self.log.debug("Created one pool with 100% of the available storage "
```

Comment:
(pylint-logging-format-interpolation) Use lazy % formatting in logging functions